Helm/opensearch-dashboards/values.yaml

```
@@ -0,0 +1,136 @@
+# Default values for opensearch-dashboards.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+opensearchHosts: "https://opensearch-cluster-master:9200"
+replicaCount: 1
+
+image: "opensearchproject/opensearch-dashboards"
+imageTag: "latest"
+imagePullPolicy: "IfNotPresent"
+
+imagePullSecrets: []
+nameOverride: ""
+fullnameOverride: ""
+
+serviceAccount:
+  # Specifies whether a service account should be created
+  create: true
+  # Annotations to add to the service account
+  annotations: {}
+  # The name of the service account to use.
+  # If not set and create is true, a name is generated using the fullname template
+  name: ""
+
+
+rbac:
+  create: true
+
+
+podAnnotations: {}
+
+extraEnvs:
+  - name: "NODE_OPTIONS"
+    value: "--max-old-space-size=1800"
```

Is this an example or necessary? 

If necessary, should be included elsewhere. Otherwise if anyone overwrites this. They lose this environment.

---

This is just an example one can use any of the kubernetes environment.

---

Maybe comment it out then? 
```suggestion
  extraEnvs: []
  # - name: "NODE_OPTIONS"
  #    value: "--max-old-space-size=1800"
```

---

We can do this. Sounds good